CHANGELOG.rst

```diff
@@ -2,6 +2,41 @@
 Change history
 ==============
 
+2.5.0
+=====
+*January, 16, 2025*
+
+**New features**
+
+* Add support for setup configuration
+    * [#293] Configuring access tokens
+    * [#294] Admin authentication via OIDC
+
+**Bugfixes/QoL**:
+
+* Add UUID to Klantinteracties admin search fields and fieldsets
+* [#254] Fix incorrect URLs being returned in API responses for ``Persoon``, ``Bijlage`` and ``CategorieRelatie``
+* [#265] Fix ``adres__icontains`` for GET requests on ``digitaleadressen`` endpoint
+* [#272] Make ``digitaalAdres.omschrijving`` not required
+* [#252] Make ``Persoon.overlijdensdatum`` optional via admin interface
+* Point help text for ``DigitaalAdres.is_standaard_adres`` to correctly cased field name
+* [maykinmedia/charts#148] Add timeouts to celery tasks
+
+
+**Project maintenance**
+
+* [#66] Update zgw-consumers to 0.35.1
```

Comment:
Should we add a warning here for the move from `APICredentials` to the `Service` model (l[ike open-zaak for example](https://github.com/open-zaak/open-zaak/blob/8a25d8d5efccda532c5ad4e4d3deef22099f03e3/CHANGELOG.rst?plain=1#L36))?

Reply:
I checked, but as far as I can see, zgw-consumers is installed, but not actually used. I don't think Open Klant does any requests to external systems as of right now